packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -36,18 +36,42 @@ enum _ScaffoldSlot {
   statusBar,
 }
 
+/// Geometry information for scaffold components.
+///
+/// To get a [ValueNotifier] for the scaffold gemoetry call
```

Comment:
(typo in geometry)

Reply:
In the fullness of time, this doc should explain what "geometry" means in this context, why you'd want to listen to this notifier, and when you'd want to handle notifications.